docs/api/alert.md

```diff
@@ -154,9 +154,15 @@ Any of the defined [CSS Custom Properties](#css-custom-properties) can be used t
 ```
 
 :::note
- If you are building an Ionic Angular app, the styles need to be added to a global stylesheet file. Read [Style Placement](#style-placement) in the Angular section below for more information.
+ If you are building an Ionic Angular app, the styles need to be added to a global stylesheet file.
 :::
 
+### Example
```

Comment:
If this example is focused around styling, can we update this heading to be "Theming" or "Styling"?